package.json

```diff
@@ -58,7 +60,7 @@
     "eslint-plugin-jsx-a11y": "^6.5.1",
     "eslint-plugin-react": "^7.27.1",
     "husky": "^7.0.4",
-    "jsdoc": "^3.6.7",
+    "jsdoc": "^4.0.2",
```

Comment:
Be careful when randomly updating packages. I would avoid doing it in this PR and revert your change here.

We usually try to update packages intentionally and look through all of the changes before updating. Especially with a new major version (3.x.x to 4.x.x, for example) there can be a lot of changes that would break some of the features we use.